metadata.rb

```diff
@@ -4,7 +4,8 @@
 license          'Apache 2.0'
 description      'provides LWRPs for interaction with AWS'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.2.1'
-depends          'build-essential', '~> 2.0'
+version          '0.3.0'
+# Last version that supports chef 12.5
+depends          'build-essential', '<= 7.03'
```

Comment:
Annoyingly you can't give a range here like

`>=2.0 <=7.03` 

Which means our options are:

1. Do what this PR does
2. Depend on ~> 5.0 (I don't know what other `build-essential` dependencies we have that would break with this)
3. Depend on `>= 2.0` (wouldn't work with Chef 12.5 past a certain version)